nuxt/pages/moderator/DraftLyricsPage.vue

```diff
@@ -0,0 +1,99 @@
+<router>
+  path: /moderator/drafts/lyrics
+  name: "moderator.drafts.lyrics"
+</router>
+
+<template>
+  <v-container class="app__section mt-4">
+    <h2>Draft Lyrics</h2>
```

Comment:
Can we just make sure the styles here are consistent with the revision history?

Reply:
I have copied and pasted all of the styles from revision history, so this should already be the case.
Marking this as resolved as I don't see any differences apart from class names (which I copy and pasted and changed names)

Reply:
I missed to copy and paste the styles from revisions page, I have done that now and is in the latest PR